Childmonitoring.txt

```diff
@@ -5,7 +5,7 @@
 LiquidCrystal lcd(12, 11, 5, 4, 3, 2);

```

Comment:
1) Naming Convention: Variable names should be descriptive and use camelCase. Consider renaming 'DHTPIN' to 'dhtPin' and 'DHTTYPE' to 'dhtType'.
2) Commenting: Add comments to explain the purpose of each variable and the overall functionality of the code.
3) Constants: Use 'const' keyword for constant values like pin numbers and sensor types. For example, 'const int dhtPin = 7;' and 'const int dhtType = DHT11;' (or appropriate DHT sensor type).
4) Functionality: The code snippet seems incomplete. It is missing necessary includes, setup() and loop() functions, and the actual reading of the sensor values.
5) Spacing: Add proper spacing and indentation to improve code readability.